rsts/deployment/cluster_config/general.rst

```diff
@@ -212,3 +212,31 @@ Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: wi
 Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: widgetmodels]** for any workflow other than ``Demand`` and any launch plan will have a tag value "critical".
 
 All other inbound CreateExecution requests will use the default values specified in the FlyteAdmin config (if any).
+
+
+Configuring K8s Pod using a Default PodTemplate
+------------------------------------------------
+
+FlytePropeller supports configuration of all K8s Pods executed using the Pod plugin. This is accomplished by creating a default PodTemplate. This functionality requires the `default-pod-template-name <https://docs.flyte.org/en/latest/deployment/cluster_config/flytepropeller_config.html#default-pod-template-name-string>`__ configuration option to be set in FlytePropeller.
+
+When executing the K8s Pods, FlytePropeller attempts to use the PodTemplate in the namespace where the Pod would be created (for example, by default, a pod in the project ``flytesnacks`` and domain ``development`` will look for a PodTemplate in the ``flytesnacks-development`` namespace). If that PodTemplate doesn't exist, FlytePropeller attempts to find it in the namespace that it runs in.
```

Comment:
I think the phrase "If that PodTemplate doesn't exist, FlytePropeller attempts to find it in the namespace that it runs in." can be a little ambiguous. In "that it runs in", the "it" here is FlytePropeller - I think it would be useful to clarify that.